mpas_analysis/sea_ice/climatology_map.py

```diff
@@ -331,6 +288,91 @@ def _compute_and_plot(self):  # {{{
                 imageDescription=imageDescription,
                 imageCaption=imageCaption)
         # }}}
+
+    def _compute_seasonal_climatologies(self):  # {{{
+
+        config = self.config
+
+        mpasMeshName = self.mpasRemapper.sourceDescriptor.meshName
+        comparisonGridName = self.mpasRemapper.destinationDescriptor.meshName
+
+        startYear = config.getint('climatology', 'startYear')
+        endYear = config.getint('climatology', 'endYear')
+
+        self.climatologyDirectory = \
+            get_mpas_climatology_dir_name(
+                config=config,
+                fieldName='seaIceAreaVol',
+                mpasMeshName=mpasMeshName)
+
+        (self.maskedClimatologyDirectory, self.remappedDirectory) = \
+            get_mpas_climatology_dir_name(
+                config=config,
+                fieldName='seaIceAreaVol_masked',
+                mpasMeshName=mpasMeshName,
+                comparisonGridName=comparisonGridName)
+
+        # the last climatology produced by NCO is always the annual, so if that
+        # exists, others are also finished.
+        climatologyFileName = \
+            '{}/mpascice_ANN_climo.nc'.format(self.climatologyDirectory)
```

Comment:
@xylar: for some reason the following is not working, in the sense that, even if the climos were computed earlier, they are recomputed again when starting over with the same settings and without purging previous results. This only happens for the sea-ice climatologies. All works as expected in the ocean climos.

Reply:
Thanks @milenaveneziani.  This happens because the sea-ice tasks don't include `ANN` in their list of seasons.  Before the `--seasons` flag in `ncclimo`, `ANN` was always produced.  Now, it's not produced if it's not explicitly needed.  I need to fix this.  Instead, this check should be making sure that *all* requested seasons are present.